windows_registry/README.md

```diff
@@ -0,0 +1,67 @@
+# Windows Registry Integration (Beta)
```

Comment:
```suggestion
# Windows Registry Integration (Beta)
```
Just an FYI. The `(beta)` in the title won't appear in the docs - they use the title from the manifest. Do we want to add a note below that the integration is in beta?

Reply:
thanks for the note! dropped the beta as it'll just go straight to generally available instead